### PR TITLE
Fix links to otst! wiki article

### PR DESCRIPTION
**Pull request by Trigonoculus**

### CODE REVIEW

File: wiki/Tournaments/fr.md [hunk 1]
```diff
@@ -222,4 +222,4 @@ Voir aussi
 [flag_TN]: /wiki/shared/flag/TN.gif
 [flag_TW]: /wiki/shared/flag/TW.gif
 [flag_US]: /wiki/shared/flag/US.gif
-[flag_VN]: /wiki/shared/flag/VN.gif
\ No newline at end of file
+[flag_VN]: /wiki/shared/flag/VN.gif
```
Comment (Krystool) [inline wiki/Tournaments/fr.md]: Not changed.

File: wiki/Tournaments/en.md [hunk 2]
```diff
@@ -222,4 +222,4 @@ See also
 [flag_TN]: /wiki/shared/flag/TN.gif
 [flag_TW]: /wiki/shared/flag/TW.gif
 [flag_US]: /wiki/shared/flag/US.gif
-[flag_VN]: /wiki/shared/flag/VN.gif
\ No newline at end of file
+[flag_VN]: /wiki/shared/flag/VN.gif
```
Comment (Krystool) [inline wiki/Tournaments/en.md]: Not changed anything.

**Comment by Trigonoculus** [comment]
@Joehuu @Krystool I used the web editor on github (quick fix), not sure what I broke.

**Comment by Joehuu** [comment]
It's fine. The web editor added a trailing line in the end, and all files should have that default.

**Comment by Krystool** [comment]
Writing access member can approve and merge this, please?

**Pull request merged**